app/views/me/notices/index.html.haml

```diff
@@ -1,12 +1,24 @@
-.h2.first=:notices.t
+%div
+  %span.h2.first=:notices.t
+  - if @notices.total_pages < 2 && @notices.any?
+    %span.h2.dissmiss-all-notices-button.dismiss-all-wrapper.pull-right
+      = dissmiss_all_notices_button
+
 - if @notices.empty?
   %ul.shaded
     %li= :none.t
 - else
-  = pagination_links @notices
+  %div
+    .pagination-wrapper.pull-left
+      = pagination_links @notices
+    - if @notices.total_pages > 1
```

Comment:
It depends on the place where we would like to place a new button / link maybe.
Current implementation place it at right corner of table, and the problem with it, that `Notices` title and pagination block have different display options and size.
So button placed differently if we use one css class for both cases..

By the way we could place it near the `Notices` title, for both cases, then it could be simplified, but it looks a bit overloaded for me then.

What about pagination: I believe that pagination block does not shown for pages less than 2, because it has no sense then.